src/amber/cli/commands.cr

```diff
@@ -16,41 +16,5 @@ class Cli::Command
 end
 
 module Amber::CLI
-  class MainCommand < ::Cli::Supercommand
```

Comment:
Can we leave this here for now as it's not related to this PR? As the number if files increase for non abstraction reasons code bases become harder to navigate.

Reply:
Will do

Reply:
Thanks :)

Reply:
Leaving this since this is relative a small change.

Reply:
This made more sense here since main command has a different namespace than the other commands.